runtime/lua/vim/secure.lua

```diff
@@ -1,4 +1,42 @@
 local M = {}
+---@private
```

Comment:
```suggestion

---@private
```

Reply:
Done in d3d5654ca469c73ed196ac902a9e1eaadf4d0ec1